pkgs/applications/misc/nemu/default.nix

```diff
@@ -0,0 +1,107 @@
+{ stdenv, config, lib, fetchFromGitHub
+, cmake
+, pkg-config
+, gettext
+, libpthreadstubs
+, libudev
```

Comment:
`libudev` is an alias to `udev`, switch to using `udev` so ofborg eval won't fail:

https://gist.github.com/GrahamcOfBorg/0f0eac15e861b506912834b1000a8133